src/icons.scss

```diff
@@ -22,7 +22,18 @@
 }
 
 .trophy-icon {
-    fill: gold !important;
     font-size: 3em !important;
     stroke: black;
+
+    &--gold {
```

Comment:
would you need important if you did multi-class?

```
& &--silver {}
```

Reply:
ya I tried that as a fix already